library/src/main/java/org/xmtp/android/library/Client.kt

```diff
@@ -69,7 +70,7 @@ class Client() {
     constructor(
         address: String,
         privateKeyBundleV1: PrivateKeyBundleV1,
-        apiClient: ApiClient
+        apiClient: ApiClient,
```

Comment:
Is the trailing comma a ktlint thing? I'm not used to them but am pro consistency so I can adjust!

Reply:
I got you I don't know why they keep creeping up. Must be a auto format thing. 🤔